Automattic/wp-calypso
# Woo Express Medium: Added Plans page

Closes https://github.com/Automattic/wp-calypso/issues/74126

Designs: jBwlERMS350NqhJNHZzAVF-fi-98%3A33334

## Proposed Changes

* Added new Plans page for WX Medium plans
* Moved WX Express Medium features list to a more appropriate path

## Testing Instructions

Desktop

* Get a site with the WX Medium plan
   * Start with a free site
   * Go to `/checkout/<site-slug>/wooexpress-medium-bundle-monthly`
   * Finish checkout
* Go to the Plans page (`/plans/<site-slug>`)
* With the feature flag enabled (`/plans/<site-slug>?flags=plans/wooexpress-medium`) you should see the new page.

Figma:
![image](https://user-images.githubusercontent.com/3801502/224869951-d4e64796-e6ca-4fb8-84da-2928a0a47dea.png)

HTML:
![image](https://user-images.githubusercontent.com/3801502/224869966-65978145-36a5-466d-b2ce-96cc145563f2.png)

Mobile
* Change the view to mobile mode

Figma: 
![image](https://user-images.githubusercontent.com/3801502/224870280-3b818bcc-a29b-4a6a-8b9e-66b9f6086f4f.png)

HTML:
![image](https://user-images.githubusercontent.com/3801502/224870305-e87c3fc6-ba3f-4a5a-91a3-ecf958f7c82b.png)
